### ISSUE TITLE
Support Advanced Post Creation add-on

### ISSUE THREAD

**Issue by rafaehlers** [GravityKit/GravityView#1268]
If the form has [post fields](https://docs.gravityforms.com/category/user-guides/post-fields/), when editing the entry in GravityView, the post created during form submission will also be modified. 

However, if the user is using the [Advanced Post Creation add-on](https://www.gravityforms.com/introducing-advanced-post-creation/), modifying the entry won't affect the post because the post was created via a feed. [More about that here.](https://docs.gravitykit.com/article/245-editable-post-fields)

I tried using the code below to [trigger the feed](https://docs.gravityview.co/article/534-trigger-feeds-edit-entry) after editing the entry and hopefully updating the post, but that didn't work. A new post was created. It seems their feed doesn't support editing.

```
add_action( 'gravityview/edit_entry/after_update', 'gravityview_trigger_feeds', 10, 3 );

function gravityview_trigger_feeds( $form = array(), $entry_id = array(), $object ) {
   
	add_filter( 'gform_is_feed_asynchronous', '__return_false', 1294873 );
  
	$Advanced_Post_Creation = GF_Advanced_Post_Creation::get_instance();
	$Advanced_Post_Creation->maybe_process_feed( $object->entry, $form );
  
	remove_filter( 'gform_is_feed_asynchronous', '__return_false', 1294873 );
    
}
```

It looks like we need a specific solution via some custom code in GravityView to handle that.

To sum it up: GravityView needs to be able to edit an entry and update the post created by the Advanced Post Creation feed at the same time.

https://support.gravitykit.com/search?f[tag]=advanced%20post%20creation

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/705881396/17588/

**Comment by soulseekah** [comment]
Rafael, I've researched the ticket a bit and am confident that the rendering issue is in GF, not GV. The checkboxes are not rendered properly by the checkbox fields in GF, the same bug is seen when trying to edit the form in the backend. It's always a good idea to look at how things are working in the GF entry editor.

I've updated the ticket with an explanation. Putting on hold meanwhile.

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/1026268043/23842/

**Comment by rafaehlers** [comment]
FYI at this moment, only this add-on is supported: https://wordpress.org/plugins/gravity-forms-custom-post-types/

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/1340823929/30641/

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/1348874516/30890/

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/1408891934/32217/

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/1403486935/32061

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/1515740997/34503/

**Comment by rafaehlers** [comment]
With this code:

```

add_action( 'gravityview/edit_entry/after_update', 'gravityview_trigger_feeds', 10, 3 );

function gravityview_trigger_feeds( $form = array(), $entry_id = array(), $object ) {
  
    if ( 215 !== (int) $form['id'] ) { // replace 100 with your form ID
		return;
	}
  
	add_filter( 'gform_is_feed_asynchronous', '__return_false', 1294873 );
  
	$Advanced_Post_Creation = GF_Advanced_Post_Creation::get_instance();
	$Advanced_Post_Creation->maybe_process_feed( $object->entry, $form );
  
	remove_filter( 'gform_is_feed_asynchronous', '__return_false', 1294873 );
    
}
```

from here: https://docs.gravityview.co/article/534-trigger-feeds-edit-entry it just creates another Post after editing an entry. It doesn't edit the existing one. 

So no good 👎

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/1618245100/36677?folderId=408730

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/1688634206/37926

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/1747402373/38917

**Comment by zackkatz** [comment]
Related to #1120 and #1352

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/1885533964/41270

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/1926145902/42012/

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/1992864977/43186/

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/2231297005/48698/

**Comment by rafaehlers** [comment]
@omarkasem can you please look into this next?

**Comment by rafaehlers** [comment]
@zackkatz, perhaps this is an interesting next one for @doekenorg instead? Another low-hanging fruit from support.

**Comment by doekenorg** [comment]
I like it. I've assigned myself. Will work on this!

**Comment by floatytheastronaut** [comment]
FreeScout #77305. <br>https://support.gravitykit.com/conversation/77305?folder_id=1

**Comment by zackkatz** [comment]
This was released in Version 2.20. Let's notify people!

**Comment by rafaehlers** [comment]
Users notified! ✅
